config.py

```diff
@@ -10,11 +10,6 @@ class Config(object):
 
 
 class TestConfig(Config):
-    TESTING = True
```

Comment:
Could you add a bit more detail to your commit message to explain why you're doing this please?

Reply:
@jonodrew I have added more detail - let me know what you think.